### ISSUE TITLE
Provide executables?

### ISSUE THREAD

**Issue by NightProgramming** [MestreLion/wavegain#9]
I think it would be nice if executables of version 1.3.2 could be provided as part of the release assets. Currently those assets only consist of the zip and tar.gz files, which is basically the same as selecting the 1.3.2 tag in GitHub then clicking on "Code" and then on "Download ZIP".

I understand that it might not be possible for every OS (e.g. Mac OS) but maybe for Windows and Linux executables could be added?

**Comment by MestreLion** [comment]
Sorry for taking so long to reply!

So... I'm not familiar with Github's tasks to automate builds... but if you give me the steps / PRs, I will gladly add the executables to the release

**Comment by NightProgramming** [comment]
Thank you for your reply! I don't think a build process is absolutely required. Executables can simply be added to a release (although this requires to have them locally on your computer first). Please see step 9 of [release documentation](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository). Also it's possible to edit past releases, so it's not necessary to create a new release for the executables. 

If you cannot get the files locally and upload them to the release, then yes, I guess a build process would be required. Unfortunately I also don't know anything about it.

**Comment by zvezdochiot** [comment]
Hi @NightProgramming .

Here are these assembler inserts:
https://github.com/MestreLion/wavegain/blob/01b7859d483cdecff02f16dbed42affc55545c73/audio.c#L165-L171
will not allow you to just get binaries for different OSes. Only for yourself. :)
 
Good luck.

**Comment by zvezdochiot** [comment]
Hi @NightProgramming .

Made assembly without "asm"s, `int64 == long long int` and `-lssp`. But testing is needed.
[wavegain-1.3.2-no_asm-llint-lssp-win32.zip](https://github.com/MestreLion/wavegain/files/10446513/wavegain-1.3.2-no_asm-llint-lssp-win32.zip)

PS: Resolved: https://github.com/Sound-Linux-More/wavegain/releases/tag/1.3.3

**Comment by MestreLion** [comment]
> If you cannot get the files locally and upload them to the release, then yes, I guess a build process would be required. Unfortunately I also don't know anything about it.

Yup, I don't have a Mac or Windows to create executables for these platforms. I guess I _could_ add my personal 32-bits self-compiled Linux executable, but I don't think this would add much (or any) value, as distro packagers have their own build infrastructure anyway, and linux users can simply type `make`.

All modern projects in Github use some of form of automated, multi-platform build recipe, I just don't know how to set one up. And, as @zvezdochiot said, the asm blobs might prevent it from working :(

**Comment by zvezdochiot** [comment]
@MestreLion say:
>the asm blobs might prevent it from working :(

By the way. What for? I do not know the subtleties, but I would like to know the difference between:
https://github.com/MestreLion/wavegain/blob/01b7859d483cdecff02f16dbed42affc55545c73/audio.c#L277
and:
```c
	mantissa = (int) (floor (in));
```
:warning: I'm not sure. It's just a question!

**Comment by MestreLion** [comment]
I have no idea either. I'm not the original author of the code, and to be very honest I've never studied the code thoroughly. I just did minimal changes  to make it compile and run in Linux

**Comment by zvezdochiot** [comment]
Hi @MestreLion .

I can't PR as I have a cleared git history. And I do not intend to change my git history ![GitHub repo size](https://img.shields.io/github/repo-size/Sound-Linux-More/wavegain) to yours ![GitHub repo size](https://img.shields.io/github/repo-size/MestreLion/wavegain). See https://github.com/Sound-Linux-More/wavegain for recipe and testing.

Current my version: [1.3.3](https://github.com/Sound-Linux-More/wavegain/releases/tag/1.3.3)

Good luck.
